knimin/lib/data_access.py

```diff
@@ -2415,6 +2415,144 @@ def set_deposited_ebi(self):
                  WHERE barcode IN %s"""
         self._con.execute(sql, [barcodes])
 
+    def create_study(self, study_id, title=None, alias=None, notes=None):
+        """Creates a study
+
+        Parameters
+        ----------
+        study_id : int
+            Positive if it is a Qiita study ID, negative if others
```

Comment:
This positive / negative system was suggested by @josenavas . @josenavas do you have any comments on this?